Source/Core/Common/BitFieldView.h

```diff
@@ -0,0 +1,581 @@
+#pragma once
```

Comment:
The legal header is missing.

Reply:
Saving that for later.  Still haven't decided if I'll apply BSD or something else.